Woof/WoofTests/Model/Preferences/PreferencesHandlerLoadTests.swift

```diff
@@ -0,0 +1,35 @@
+import XCTest
+
+final class PreferencesHandlerLoadTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        PreferencesHandlerTestHelper.cleanStorage()
+    }
+
+    override func tearDown() {
+        PreferencesHandlerTestHelper.cleanStorage()
+        super.tearDown()
+    }
+
+    func testLoadSelectedRoleExistInAPI() {
+        _ = PreferencesHandler.loadSelectedRole()
+    }
+
+    func testLoadSelectedRoleReturnsExpectedRole() {
+        _ = PreferencesHandler.saveSelectedRole(.owner)
+
+        XCTAssertEqual(PreferencesHandler.loadSelectedRole(), .owner)
+    }
+
+    func testLoadSelectedRoleReturnsDefaultCaseForNonExistingKey() {
+        XCTAssertEqual(PreferencesHandler.loadSelectedRole(), .none)
+    }
+
+    func testLoadSelectedRoleReturnsExpectedRoleWhenRewrittenByNonDefaultCase() {
+        let rewrittenRole: Role = .sitter
```

Comment:
nit:

```suggestion
        let overriddenRole: Role = .sitter
```

Reply:
thx, i'll fix it

Reply:
I reworked the code and change this function